components/View/VirtualizedListView/index.js

```diff
@@ -158,6 +159,10 @@ class VirtualizedListView extends React.Component {
                 itemsPerPage,
                 itemHeight,
             });
+
+            if (onItemHeight) {
+                onItemHeight(itemHeight);
```

Comment:
Is this for returning the item height?
If so, lets rename it to `onItemHeightSet` as `onItemHeight` doesn't make sense.